ioc_fanger/ioc_fanger.py

```diff
@@ -31,12 +31,13 @@ def fang(text: str, debug=False):
 @click.argument("text", required=False)
 def cli_fang(text):
     """CLI interface for fanging indicators."""
-    stdin_text = click.get_text_stream("stdin")
-
+    
     if text:
         fanged_text = fang(text)
         print(fanged_text)
-    elif stdin_text:
+        return
```

Comment:
I recommend adding a newline here so it is easier for readers to see the two different paths we can take in this function (one path if there is text and another if there is stdin_text):

```suggestion
        return

```